lib/rules/comment-empty-line-before/README.md

```diff
@@ -140,7 +140,7 @@ a {
 
 #### `"stylelint-commands"`
 
-Ignore comments that deliver commands to stylelint, e.g. `/* stylelint-disable color-no-hex */`.
+Ignore configuration comments, e.g. `/* stylelint-disable color-no-hex */`.
```

Comment:
I hadn't realised we had an option related to commands. I think we're ok to move forward with "configuration comment", though, as we have a few other rule secondary options which are at odds with our conventions.